components/browser/session-storage/src/main/java/mozilla/components/browser/session/storage/serialize/BrowserStateWriter.kt

```diff
@@ -91,8 +91,14 @@ private fun JsonWriter.tab(
         name(Keys.SESSION_LAST_ACCESS)
         value(tab.lastAccess)
 
+        name(Keys.SESSION_LAST_MEDIA_URL)
+        value(tab.lastMediaAccessState.lastMediaUrl)
```

Comment:
Should we only persist the url if we're active, similar to the reader logic below (line 103). This would work because you have a default value (empty string) anyway for the constructor, right?

Reply:
There's also a `BrowserStateWriterReaderTest.kt ` we should update.

Reply:
> Should we only persist the url if we're active, similar to the reader logic below (line 103). This would work because you have a default value (empty string) anyway for the constructor, right?

I like the idea but I'm not sure what "if we're active" means.
The `TabSessionState.hasMediaPlayed()` is not available atm but I see `:feature-tabs` as a dependency for tests so maybe we can add that for code also and use this check?

Reply:
OK I doubt it's worth it. Let's leave it :)